CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 7.4.0
```

Comment:
think the previous shutdown behavior is a bug, not a feature. If someone asks logstash to shutdown, and we give up retrying because we're shutting down, that is the bug.